components/rich_text_editor/rich_text_editor.vue

```diff
@@ -102,6 +103,14 @@ export default {
       type: [Boolean, Object],
       default: false,
     },
+
+    /**
+     * Placeholder text
+     */
+    placeholder: {
+      type: String,
+      default: 'Type here...',
```

Comment:
Hmm, now I'm thinking that probably this should be left empty by default as maybe we don't always want to show it and also since the default is in English only.

Reply:
i agree. changed